vividus/src/main/java/org/vividus/report/MetadataLogger.java

```diff
@@ -54,7 +54,7 @@
     private static final String NEW_LINE = "%n";
     private static final Logger LOGGER = LoggerFactory.getLogger(MetadataLogger.class);
     private static final Pattern SECURE_KEY_PATTERN = Pattern
-            .compile(".*(password|access-?key|api-?key|secret|token).*", Pattern.CASE_INSENSITIVE);
+            .compile(".*(password|access-?key|api-?key|secret|token|private-?key).*", Pattern.CASE_INSENSITIVE);
```

Comment:
what about `key` ?

Reply:
I can add if really needed. Please, provide usage examples?

Reply:
The idea is to have a generic matcher for anything with the `key` part

Reply:
The question is do you really want to hide everithing with `key` part?
Looks like lot of confings could be affected, e.g.: https://docs.vividus.dev/vividus/latest/plugins/plugin-azure-cosmos-db.html#_configuration